src/viewer.js

```diff
@@ -411,11 +411,12 @@ const Viewer = function Viewer(targetOption, options = {}) {
         const featureId = urlParams.feature;
         const layerName = featureId.split('.')[0];
         const layer = getLayer(layerName);
-        const clusterSource = layer.getSource().source;
+        const layer2 = getLayer('intressepunkt');
```

Comment:
?

Reply:
That line maybe ain't necessary....removed!